dateparser_cli/fasttext_manager.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+import urllib.request
+
+from .utils import date_parser_model_home
+
+
+def fasttext_downloader(model=[]):
```

Comment:
Why is ``model`` a list? Also if it needs to be a list, mutable default arguments are normally not a good idea, see for example https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

Reply:
Thanks for the resource

Reply:
I tried to make some changes, please suggest it that's fine

Reply:
@gavishpoddar did you push the changes? Could you please send a link to the commit you have in mind?

Reply:
Hi @lopuhin, not I have committed and pushed the changes.

Reply:
@gavishpoddar just started the workflow run